lib/Fhp/Segment/PaginateableTrait.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Fhp\Segment;
+
+trait PaginateableTrait
```

Comment:
Ich würde den Trait weglassen und den Setter einfach in alle Klassen kopieren. Visuell spart der Trait dort nur eine einzige Zeile Code, was den Aufwand nicht wert ist. Und von der Komplexität her ist ein Setter so simpel, dass es nicht schadet, ihn an mehreren Stellen zu implementieren.

Dafür wird es dann in den Segment-Klassen offensichtlicher, wer das Feld setzt. Und es muss nicht überall zwangsläufig `aufsetzpunkt` heißen.

Reply:
Klingt gut.